VectorRumble.Shared/Gameplay/ShipManager.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Content.Pipeline.Serialization.Intermediate;
```

Comment:
@CartBlanche its general bad practice to use this at runtime. The main reason is this does NOT work on Mobile or Consoles. It will only work on Desktop. The PCL version of the Pipeline assembly is a Facade assembly , i.e it has NO implementation so it will throw a `NotImplemented` exception on Non Desktop platforms. 

So your options are either, use the Pipeline Xml Processor to use the default .net XmlSerializer.

Reply:
Why isn't this supported/implemented on all platforms? It was part of XNA since version 3, I think. The issue is that the default XMLSerializer is *shit* in comparison to the XNA one. Hence why I used it the XNA one.